database/src/main/java/entities/Genre.java

```diff
@@ -0,0 +1,25 @@
+package entities;
+
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.io.Serializable;
+
+@Entity
+@NoArgsConstructor
+public class Genre implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
```

Comment:
I want to know if the field is private and there is no getter, Is hibernate able to read the state of the genre object ?